### ISSUE TITLE
[Discovery] Focus is not visible in Breadcrumbs.

### ISSUE THREAD

**Issue by wlla** [NYPL/design-toolkit#208]
Am running through Discovery on QA—most recent updates. Focus is not visible for links in Breadcrumbs. They links get focus, just not visible.
